publish-seven-things-application (#2103)
DFE-Digital/get-into-teaching-app: app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md

---
title: 7 things to include in your teacher training application
date: "2021-12-16"
author: Juliet Fern, Mieka Harris and Kate Hudson-Glynn
images:
  bedfordshire:
    path: "media/images/content/blog/bedfordshire.jpg"
    thumbnail_path: "media/images/content/blog/thumbnails/bedfordshire-thumbnail.jpg"
    alt: "Teacher training students sit at desks with a number of resources such as paper, games and dice"
description: |-
   Juliet Fern, Mieka Harris and Kate Hudson-Glynn from the University of Bedfordshire share seven key things to include in your teacher training application.
keywords:
  - applications
  - personal statements
tags:
  - applications
  - becoming a teacher
  - teacher training
  - personal statements
---

$bedfordshire$

_Not sure what to include in your teacher training application? Just think of the word TEACHER! Juliet Fern, Mieka Harris and Kate Hudson-Glynn from the University of Bedfordshire explain all._

> So you want to be a teacher — why is this? Were you taught by someone amazing? Do you love a specific subject? Do you want to give something back? Do you like working with others? Do you like to learn new things?

Reviewer: ```suggestion
So you want to be a teacher — why? Were you taught by someone amazing? Do you love a specific subject? Do you want to give something back?
```
Think just sticking with three examples reads a bit better

> Many people who join the teaching profession want to inspire the next generation in some way. However getting this across in your application and interview can be tricky. Remember that teacher training providers are looking for your potential to be a teacher, not just what you have already done. It is also really important to get across a feeling of positive energy about your goals and aspirations. Although different training providers will adopt different approaches to the interview process, all are looking for a self-motivated, passionate individual.

Reviewer: ```suggestion
Many people who join the teaching profession want to inspire the next generation in some way. However, getting this across in your application and interview can be tricky. Remember that teacher training providers are looking for your potential to be a teacher, not just what you have already done. It is also really important to get across a feeling of positive energy about your goals and aspirations. Although different training providers will adopt different approaches to the interview process, all are looking for a self-motivated, passionate individual.
```

You can use the acronym TEACHER to help you present yourself at your very best in your application and interview.

## Thought

What reflections do you have on your experiences of learning and of supporting others to learn? Think about your own learning in and out of school, how you have helped others to improve at something and whether you could have done anything differently.

## Experience

Why do you want to become a teacher? Consider the experiences that have inspired you to apply for teacher training.

## Aptitude

> What skills do you have that will make you a good teacher? Think about the qualities your favourite teachers have and research the classroom practice of being a great teacher.

Reviewer: ```suggestion
What skills do you have that will make you a good teacher? Think about the qualities your favourite teachers had and research the classroom practice of being a great teacher.
```

## Collaboration

How do you establish rapport with others to achieve shared goals? Work out how you can demonstrate that you work well with people of different ages, backgrounds and levels of experience.

## Heartfelt

What qualities do you have that will make you a good teacher? Don’t forget your softer skills, such as patience, kindness, approachability and empathy.

## Enthusiasm

Why are you passionate about joining the teaching profession? Come up with your top three reasons why this is the career for you.

## Resilience

What have you found challenging and how did you deal with it? Find an example of a challenge you’ve overcome, and explain what you learnt and how you can apply this to future challenges.

> Have confidence in yourself and remember that training providers are on your side! We always enjoy reading about your experiences from school, family, jobs and volunteering, and look forward to meeting you at interview where you can demonstrate further how well-informed and interested you are in becoming a TEACHER.

Reviewer: ```suggestion
Have confidence in yourself and remember that training providers are on your side! We always enjoy reading about your experiences from school, family, jobs and volunteering. We look forward to meeting you at interview where you can show us how well-informed and interested you are in becoming a TEACHER.
```
Just split up the sentences and changed a bit for readability

> > Whatever stage you’re at with your application, our expert advisers can help. [Sign up for a Teacher Training Adviser](https://adviser-getintoteaching.education.gov.uk/) who can give you free, one-to-one support.

Reviewer 2: ```suggestion
Whatever stage you’re at with your application, our expert advisers can help. [Sign up for a Teacher Training Adviser](https://adviser-getintoteaching.education.gov.uk/) who can give you free, one-to-one support.
```

@emilytraff - now the last paragraph will always be styled as a summary so this doesn't need to be quoted. PR is spot on other than that though 👍🏽
